linetools/lists/linelist.py

```diff
@@ -344,7 +349,195 @@ def all_transitions(self,line):
             else:
                 raise ValueError('Line {} not found in the linelist'.format(line))
 
+    def strongest_transitions(self,line, wvlims, z, n_max=3,verbose=False):
+        """For a given single line transition, this function returns
+        the n_max strongest transitions of the ion found in 
+        the linelist, within the wavelenght range wlims at a given redshift z
+
+
+        Parameters:
+        ----------
+        line: str or Quantity
+            Name of line. (e.g. 'HI 1215', 'HI', 'CIII', 'SiII', 1215.6700*u.AA)
+            [Note: when string contains spaces it only considers the first
+             part of it, so 'HI' and 'HI 1215' and 'HI 1025' are all equivalent]
+            [Note: to retrieve an unknown line use string 'unknown']
+        wvlims : tuple of Quantity
```

Comment:
Why have this be in observed space?
I think the input should be in rest wavelength.
Leave it to the user to divide by (1+z)